domains/lmckh.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "llwhqp-tfqvr",
+    "email": "veronicaaspasia@gmail.com"
+  },
+  "records": {
+    "CNAME": "lkhzzb.czgrh.pages.dev"
+  },
+  "proxied": true
```

you cant proxy cf pages
```suggestion
  }
```